test/lsrc-spaces.t

```diff
@@ -0,0 +1,8 @@
+  $ . "$TESTDIR/helper.sh"
+
+Should handle dotfiles with spaces
+
+  $ touch ".dotfiles/sublime text 3.config"
```

Comment:
Maybe we should have an example that has an `'a'` in the name. This wouldn't really do a good job of finding encoding/decoding errors arising from mishandling of the control character.